src/main/java/sirius/biz/model/PersonData.java

```diff
@@ -153,6 +154,14 @@ public int hashCode() {
         return Objects.hash(title, salutation, firstname, lastname);
     }
 
+    @BeforeSave
+    protected void checkSalutation() {
+        if (Strings.isFilled(salutation)) {
+            // a HandledException will be thrown if the value is not contained in the codelist
+            codeLists.getRequiredValue("salutations", salutation);
```

Comment:
Wouldn’t it be nicer to create a verify value in codelists so that an exception is thown on purpose and not as side effect? The code should already be available anyway(?)

Reply:
Done